src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -25,6 +25,9 @@
 
     <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
       <File Include="@(LibrariesRuntimeFiles)" TargetPath="framework" />
+      <!-- exclude shared libs as we only need static ones -->
+      <LibPackageExcludes Include="framework\%2A%2A\%2A$(LibSuffix)"/>
```

Comment:
@am11 I think this is breaking in https://github.com/dotnet/sdk/pull/44751, we see lots of failures like

> C:\h\w\9AF40878\t\dotnetSdkTests\ybuooc24.wa2\.nuget\packages\microsoft.dotnet.ilcompiler\10.0.0-alpha.1.24558.9\build\Microsoft.NETCore.Native.Publish.targets(78,5): error : The FrameworkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative 

I think this is because we're now excluding .dll on Windows since that is the LibSuffix there but that also means no managed assemblies are included anymore.

Reply:
Fixed https://github.com/dotnet/runtime/pull/109676. We only need to exclude Native.dll etc.